cuda-flags.file

```diff
@@ -0,0 +1,32 @@
+# file: cuda-flags
+# define the CUDA compilation flags in a way that can be shared by SCRAM-based and regular tools
+
+%define cuda_flags -std=c++14 -O2
+
+# imply __host__, __device__ attributes in constexpr functions
+%define cuda_flags %cuda_flags --expt-relaxed-constexpr
```

Comment:
@smuzaffar looks like I cannot use recursive definition of the flags.

Is there some way around it ?

Reply:
Shahzad is away until 9th of march and I can't answer straight away, does rpm complains or it doesn't expand it as expected ?

Reply:
`rpmbuild` complains about some sort of "recursive macro definition" (sorry, I didn't copy the error message).

For the moment, I'll use N different macros and merge them at the end.